pkg/apis/pipeline/v1/pipelinerun_validation.go

```diff
@@ -109,6 +109,10 @@ func (ps *PipelineRunSpec) Validate(ctx context.Context) (errs *apis.FieldError)
 		errs = errs.Also(validateTaskRunSpec(ctx, trs).ViaIndex(idx).ViaField("taskRunSpecs"))
 	}
 
+	if ps.TaskRunTemplate.PodTemplate != nil {
```

Comment:
will this result in a nil pointer error if TaskRunTemplate is nil? Do we have tests covering this case?

Reply:
`TaskRunTemplate` is not a pointer in v1 so I don't think it is a problem?

Reply:
ah you're right-- we probably should have made it a pointer but oh well